test/optim/test_optim.py

```diff
@@ -27,6 +27,9 @@
     skipIfRocm,
     skipIfTorchDynamo
 )
+
+from torch._dynamo import disable
```

Comment:
Nit (`@disable` decorator looks a bit scary, `@disable_dynamo` less so ;) )
```suggestion
from torch._dynamo import disable as disable_dynamo
```